src/components/routes/retrohunt/components/model.tsx

```diff
@@ -0,0 +1,57 @@
+export type RetrohuntParamProps = {};
+
+export type RetrohuntPageType = 'drawer' | 'page';
+
+export type RetrohuntPost = {
+  description: string;
+  classification: string;
+  archive_only: boolean;
+  yara_signature: string;
+};
+
+type Retrohunt = {
+  code: any;
+  creator: any;
+  tags: any;
+  description: any;
+  created: any;
+  classification: any;
+  yara_signature: any;
+  raw_query: any;
+  total_indices: any;
+  pending_indices: any;
+  pending_candidates: any;
+  errors: any;
+  hits: any;
+  finished: any;
+  truncated: any;
+  archive_only?: boolean;
+};
+
+export const DEFAULT_RETROHUNT: Retrohunt = {
```

Comment:
There is a lot of default values that should not exist here

Reply:
I removed all the sub-files that I had before into one concise file that has all the duplicated values removed.